src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -51,10 +47,10 @@ interface FinderAttributes {
   type: string;
 }
 
-interface SavedObjectFinderItem extends SavedObject {
+interface SavedObjectFinderItem extends SavedObjectCommon {
   title: string | null;
   name: string | null;
-  simple: SimpleSavedObject<FinderAttributes>;
+  simple: SavedObjectCommon<FinderAttributes>;
```

Comment:
It looks like we have `FinderAttributes` here and in `src/plugins/saved_objects_finder/common/types.ts` now. Should we remove the one in this file and import from `common` instead?